docs/management/advanced-options.asciidoc

```diff
@@ -258,8 +258,12 @@ Highlights results in *Discover* and saved searches on dashboards. Highlighting
 slows requests when working on big documents.
 
 [[doctable-legacy]]`doc_table:legacy`:: 
-Control the way the Discover's table looks and works. Set this property to `true` to revert to the legacy implementation.
+Controls the way the document table looks and works. Set this property to `true` to revert to the legacy implementation.
 
+[[discover-searchFieldsFromSource]]`discover:searchFieldsFromSource`::
+When enabled, documents will be loaded and displayed using `_source`. This is soon going to be deprecated. 
+When disabled, the new `fields` parameter of Elasticsearch search service is being used. This way runtime, alias and 
+multi fields can be displayed.
```

Comment:
We can future-proof and simplify this a bit by linking to the Elasticsearch docs:
```suggestion
[[discover-searchFieldsFromSource]]`discover:searchFieldsFromSource`::
Load fields from the original JSON {ref}/mapping-source-field.html[`_source`].
By default, Discover loads fields using the {es} search API's
{ref}/search-fields.html#search-fields-param[`fields`] parameter.
```

As @kertal mentioned, I think we should align the UI text with these docs.

Reply:
yes, that's very useful this way, just one think, what's default depends on which version currently, default in master is different than in 7.12, so the text has to cover both

Reply:
Thanks @kertal. I've updated the suggestion to remove the "default" copy.

Reply:
thanks @jrodewig.  I added one small tweak to put Discover in bold.

Reply:
@thanks both,  btw hi @jrodewig, it's been a while since Mountain View! Hope you're fine!